kernel/src/expressions/mod.rs

```diff
@@ -280,6 +280,11 @@ impl Expression {
         Self::unary(UnaryOperator::IsNull, self)
     }
 
+    /// Create a new expression `self IS NOT NULL`
+    pub fn is_not_null(self) -> Self {
```

Comment:
I got tired of writing the full version all over and finally added this helper that we probably should have had all along.